py/torch_tensorrt/dynamo/_settings.py

```diff
@@ -143,6 +144,16 @@ class CompilationSettings:
     use_distributed_mode_trace: bool = USE_DISTRIBUTED_MODE_TRACE
     offload_module_to_cpu: bool = OFFLOAD_MODULE_TO_CPU
 
+    def __getstate__(self) -> dict[str, Any]:
+        state = self.__dict__.copy()
+        state["torch_executed_ops"] = {str(op) for op in state["torch_executed_ops"]}
+        return state
+
+    def __setstate__(self, state: dict[str, Any]) -> None:
+        self.__dict__.update(state)
+        ops_str = self.torch_executed_ops
+        self.torch_executed_ops = {getattr(torch.ops, op) for op in ops_str}
```

Comment:
so the stored version will have the trimmed version? Like if `torch_executed_ops = [torch.ops.aten.add]` the stored version is `aten.add`? 

In the case of the input being a string, like if `torch_executed_ops = ["torch.ops.aten.add"]` would it just get converted back a op?

Reply:
Would the string need to be trimmed?